openpype/modules/ftrack/plugins/publish/integrate_ftrack_instances.py

```diff
@@ -168,7 +168,31 @@ def process(self, instance):
         # Change asset name of each new component for review
         is_first_review_repre = True
         not_first_components = []
+        extended_asset_name = False
         for repre in review_representations:
+            # Create copy of base comp item and append it
+            review_item = copy.deepcopy(base_component_item)
+
+            # condition for multiple reviewable representations
+            # expand name to better label componenst
+            if is_first_review_repre and len(review_representations) > 1:
```

Comment:
But we don't want to add representation name for first review. That's the whole logic of `is_first_review_repre` attribute.

Reply:
well it is the point of view. I would like to have those differentiated as it makes sense to me. From your logic it doesn't make sense. Now who is right to decide what is right, right?

Reply:
I didn't, @mkolar did.

Reply:
I can offer condition with Nuke host only.

Reply:
Unfortunately, this adds a new use case to this functionality, but the use case is totally relevant. Limiting it to one Nuke is not very systematic and will explode in our faces at some point I'm afraid. 

The reason why we have the first review as the main one is to keep some consistency between multiple versions and because these extra ftrack version don't map very well to our database structure. Ftrack doesn't allow for multiple reviewable components (our representations), so we have to hack it and create "related" versions. 

Seeing this new usecase, I'd say it's reasonable to have it explicit in all cases, but I'm worried of the implication it might have if someone does this for a single shot only. Suddenly one shot has a different name format than the rest...

I think that @tokejepsen is using this in production as well. What do you think?

Reply:
:The issue is that you'll loose version progress of the subset on ftrack if you'll add representation name and there is none that has only subset name

I assume you mean, in case where v03 was published with 2 reviewable representations, but v04, suddenly only has 1 reviewable repre?

Reply:
Yes, in that case you'll have for example:
- version 3 reviews on `renderCompositingMain_mov` and `renderCompositingMain_mov_lut`
- version 4 review on `renderCompositingMain`

And related version would be empty (or with gaps in versions)
![image](https://user-images.githubusercontent.com/43494761/161275676-c099bf2f-2e56-4189-80e0-b11e1d8c1ad6.png)

Reply:
I will remove my change if you feel it is important to remain it consistently broken. But it is rather step back then forward.

Reply:
Another option would be to let users to control this behaviour from Settings. Would taht be option for you?